ts/output/svg/Wrappers/mtr.ts

```diff
@@ -107,13 +107,14 @@ CommonMtrMixin<SVGmtd<any, any, any>, SVGConstructor<any, any, any>>(SVGWrapper)
     const cSpace = this.parent.getColumnHalfSpacing();
     const cLines = [this.parent.fLine, ...this.parent.cLines, this.parent.fLine];
     const cWidth = this.parent.getComputedWidths();
+    const scale = this.getBBox().rscale;
     let x = cLines[0];
     for (let i = 0; i < this.numCells; i++) {
       const child = this.getChild(i);
       child.toSVG(svg);
       x += this.placeCell(child, {
-        x: x, y: 0, lSpace: cSpace[i], rSpace: cSpace[i + 1], w: cWidth[i],
-        lLine: cLines[i], rLine: cLines[i + 1]
+        x: x, y: 0, lSpace: cSpace[i] / scale, rSpace: cSpace[i + 1] / scale, w: cWidth[i] / scale,
+        lLine: cLines[i] / scale, rLine: cLines[i + 1] / scale
```

Comment:
A lot of division, that's expensive. Can we have `let scale = 1 / ...` and then multiply with `scale` instead?
I don't think that will introduce unwanted rounding errors.